src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -234,6 +234,18 @@ inline float avx2_load_f32(const int8_t* data, ov::element::Type type) {
     }
 }
 
+inline __m256 u4_avx2_load_f32(const uint8_t* data) {
+    float unpacked[8];
+    for (int i = 0; i < 8; ++i) {
+        uint8_t u4_element = lo4(data[i]);
+        unpacked[i] = static_cast<float>(u4_element);
+    }
+
+    __m256 result = _mm256_loadu_ps(unpacked);
+
+    return result;
+}
```

Comment:
The previous `avx2_load*` functions defined here load a single value to the vector. Here you actually load a vector to a vector? I believe it should be named different in this case.